### PR TITLE
751 element delete

### PR DESCRIPTION
**Pull request by man-git007**

### CODE REVIEW

File: app/Http/Controllers/Api/ActivityController.php [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+/**
+ * Class ActivityController.
+ */
+class ActivityController extends Controller
+{
+    private ActivityService $activityService;
+
+    /**
+     * @param ActivityService $activityService
```
Comment (sriza) [inline app/Http/Controllers/Api/ActivityController.php:22]: This phpdocs should be above $activityService variable

File: app/Http/Middleware/AccessibleRoute.php [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+/**
+ * Class AccessibleRoute.
+ */
+class AccessibleRoute
+{
+    public function handle(Request $request, Closure $next): mixed
+    {
+        $accessibleRoutes = [
+            'iati_identifier',
```
Comment (sriza) [inline app/Http/Middleware/AccessibleRoute.php:16]: iati_identifier should not be deleted

File: resources/assets/js/scripts/script.ts [hunk 3]
```diff
@@ -3,6 +3,9 @@ import jQuery from 'jquery';
 import 'select2';
 
 jQuery(function () {
+  $(function () {
```
Comment (sriza) [inline resources/assets/js/scripts/script.ts:6]: this code is probably unused.

**Review by sriza — CHANGES_REQUESTED** [review]

File: resources/assets/js/scripts/script.ts (hunk 3 above)
Comment (man-git007) [reply resources/assets/js/scripts/script.ts:6]: @titungdup can you look into this?

File: app/Http/Controllers/Admin/Activity/TransactionController.php [hunk 4]
```diff
@@ -239,6 +239,7 @@ public function update(TransactionRequest $request, $activityId, $transactionId)
                 'Activity transaction updated successfully.'
             );
         } catch (\Exception $e) {
+            dd($e->getMessage());
```
Comment (sriza) [inline app/Http/Controllers/Admin/Activity/TransactionController.php:242]: remove dd

**Review by sriza — APPROVED** [review]

**Pull request merged**